scripts/lib.sh

```diff
@@ -113,6 +113,11 @@ install-py-dev() {
         # This saves having to install postgresql on the Developer's machine + using flags
         # https://github.com/psycopg/psycopg2/issues/1286
         pip install https://storage.googleapis.com/python-arm64-wheels/psycopg2_binary-2.8.6-cp38-cp38-macosx_11_0_arm64.whl
+        BREW_PATH="$(brew --prefix)/opt"
+        # The CPATH is needed for confluent-kakfa --> https://github.com/confluentinc/confluent-kafka-python/issues/1190
+        export CPATH="${BREW_PATH}/librdkafka/include"
```

Comment:
_nit_: If you inline `$(brew --prefix librdkafka)` here, it also checks whether the package is still installed and you could even catch it down to print a nice error message.

Reply:
I like it! Thanks!